src/update-docker-tags.ts

```diff
@@ -111,17 +133,37 @@ function findTrackables(
 
 async function checkTagsAgainstArtifactRegistryAndModifyScalars(
   trackables: Trackable[],
+  lineCounter: yaml.LineCounter,
   dockerRegistryClient: DockerRegistryClient,
   logger: PrefixingLogger,
 ): Promise<void> {
   for (const trackable of trackables) {
     const prefix = `${trackable.trackMutableTag}---`;
 
     const equivalentTags = (
-      await dockerRegistryClient.getAllEquivalentTags({
-        dockerImageRepository: trackable.dockerImageRepository,
-        tag: trackable.trackMutableTag,
-      })
+      await (async () => {
+        try {
+          return await dockerRegistryClient.getAllEquivalentTags({
+            dockerImageRepository: trackable.dockerImageRepository,
+            tag: trackable.trackMutableTag,
+          });
+        } catch (e) {
+          if (e instanceof Error) {
+            let message = e.message;
+            if (e.message === `5 NOT_FOUND: Requested entity was not found.`) {
+              message =
+                'Docker image not found check the image name or tracking tags. ' +
+                'If this is a PR check that the docker image has been succesful built at least once for the PR';
```

Comment:
```suggestion
                'If this is a PR, check that the docker image has been successfully built at least once for the PR.';
```

that said we could also just check to see if trackMutableTag starts with `pr-` instead of asking the reader to do the conditional in their head :)